src/styles/block/header.scss

```diff
@@ -0,0 +1,67 @@
+body {
+  font-family: "Roboto", Arial, sans-serif;
+  background-color: #fff;
+  font-weight: 400;
+}
+
+.header {
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  display: flex;
+  justify-content: space-between;
+  padding-left: 50px;
+  padding-right: 50px;
+
+  .header__logo {
+    margin-top: 10px;
+  }
+
+  .header__img {
+    width: 40px;
+    height: 40px;
+  }
+
+  .header__list {
+    display: flex;
+    padding: 0;
+  }
+
+  .header__item {
+    list-style-type: none;
+  }
+
+  .header__list .header__item:not(:last-child) {
+    margin-right: 20px;
+  }
+
+  .header__link {
+    text-decoration: none;
+    text-transform: uppercase;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 14px;
+    height: 60px;
+    display: flex;
+    align-items: center;
+    text-align: center;
+    color: #000;
+    cursor: pointer;
+    transition: color 0.3s ease;
+    position: relative;
+  }
+}
+
+.header__link:hover,
+.is-active {
```

Comment:
Why you didn't use `sass`?
Don't write `header` again and again, just use `&__`